composer.json

```diff
@@ -13,11 +13,11 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
-        "doctrine/orm": "^2.11",
-        "giadc/json-api-request": "~3.0.3",
-        "giadc/json-api-response": "~3.0.0",
-        "symfony/cache": "^6.0"
+        "php": ">=8.2",
+        "doctrine/orm": "^2.14",
```

Comment:
I wonder when/if `laravel/doctrine` is going to move on to doctrime/orm:3

Reply:
they market themselves as `A drop-in Doctrine ORM 2 implementation for Laravel 5+ and Lumen ` So i am going to assume never.. I am just happy it gets updated at this point.